osu.Game/Screens/OnlinePlay/Components/RoomManager.cs

```diff
@@ -84,8 +84,11 @@ public virtual void CreateRoom(Room room, Action<Room> onSuccess = null, Action<
 
         private JoinRoomRequest currentJoinRoomRequest;
 
-        public virtual void JoinRoom(Room room, Action<Room> onSuccess = null, Action<string> onError = null)
+        public virtual void JoinRoom(Room room, string password = null, Action<Room> onSuccess = null, Action<string> onError = null)
         {
+            // todo: send into JoinRoomRequest directly?
+            room.Password.Value = password;
```

Comment:
Is this something to do here, or will it be a follow-up thing (if at all)?

Reply:
Missed this one, should be fixed now.